dev-db/percona-xtrabackup/percona-xtrabackup-2.4.13.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils flag-o-matic
+
+DESCRIPTION="Hot backup utility for MySQL based servers"
+HOMEPAGE="https://www.percona.com/software/mysql-database/percona-xtrabackup"
+SRC_URI="https://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-${PV}/source/tarball/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="
+	app-arch/lz4:0=
+	app-editors/vim-core
+	<dev-libs/boost-1.65.0:=
```

Comment:
Is this still the case? I do see comment below about bundled boost and libedit, but are you 100 % sure about this?

Reply:
Unfortunately yes, I thought the build failure is due to gcc 8, but it's >boost-1.63.0 really :(